### ISSUE TITLE
Crash on updating repo from user with no email

### ISSUE THREAD

**Issue by za3k** [clockfort/GitHub-Backup#15]
I called github-backup with the following:

    github@deadtree:~$ GitHub-Backup/github-backup.py -m -t <censored> vanceza /home/github/vanceza

And got this response:
```
Processing repo: freethenation/ascii-dog
Repo already exists, let's try to update it instead
Traceback (most recent call last):
  File "GitHub-Backup/github-backup.py", line 121, in <module>
    main()
  File "GitHub-Backup/github-backup.py", line 46, in main
    process_repo(repo, args)
  File "GitHub-Backup/github-backup.py", line 83, in process_repo
    update_repo(repo, dir, args)
  File "GitHub-Backup/github-backup.py", line 109, in update_repo
    os.system("git config --local gitweb.owner %s"%(shell_escape("%s <%s>"%(repo.user.name, repo.user.email.encode("utf-8"))),))
AttributeError: 'NoneType' object has no attribute 'encode'
```

This is because a github user is not guaranteed to have a known email.

**Comment by Enrico204** [comment]
@za3k I made a PR at #23 which fixes this issue

**Comment by clockfort** [comment]
Fixed in #23
